pywbem/mof_compiler.py

```diff
@@ -1897,8 +1897,15 @@ def DeleteQualifier(self, *args, **kwargs):
 class MOFWBEMConnection(BaseRepositoryConnection):
     """
     A CIM repository connection to an in-memory repository on top of an
-    underlying repository, that is used by the MOF compiler to provide rollback
-    support.
+    underlying repository API, that is used by the MOF compiler to provide
+    rollback support.  It only implementats the BaseRepositoryConnection
+    methods required to implement the mof compiler rollback functionality and a
+    minimal repository for tests within the pywbem tests suite..
+
+    MOFWBEMConnection gets CIM objects from a repository defined
+    as the in-memory repository if the constructor parameter conn is None
+    or, if conn is defined, from the repository defined by conn.  It always
+    writes qualifiers, instances, and classes to the in-memory repository.
```

Comment:
I think this is mostly stated by the proposed new text.

Reply:
DONE. Paragraph gone